src/main/scala/mesosphere/marathon/MarathonModule.scala

```diff
@@ -49,6 +51,13 @@ object ModuleNames {
   final val NAMED_SERVER_SET_PATH = "SERVER_SET_PATH"
   final val NAMED_SERIALIZE_GROUP_UPDATES = "SERIALIZE_GROUP_UPDATES"
   final val NAMED_HTTP_EVENT_STREAM = "HTTP_EVENT_STREAM"
+
+  final val NAMED_STORE_APP = "AppStore"
```

Comment:
ModuleNames.NAMED_STORE_APP is quite verbose. IMHO ModuleNames.STORE_APP or Named.STORE_APP would be OK since it's only in local scope of that file. Benefit of a rename would be shorter and more readable lines where the vals are used in annotations.